plugins/com.kronosafe.osd/model/osd.html

```diff
@@ -1,5 +1,5 @@
 <!-- This file was created using the HTML documentation generator. -->
-<!-- Creation date: Tue May 14 13:13:23 CEST 2019-->
+<!-- Creation date: Wed Jun 05 11:31:07 CEST 2019-->
```

Comment:
If I understand correctly, this file is generated from the contents of the `osd.ecore` file. Is it expected to commit this file to the git repository? Can it be removed from the repository or is it crucial to keep it here?

Reply:
We need for partnership purpose for example the html documentation without building a new plugin.